src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -870,7 +870,7 @@ private void PartialEventChecks(SourceEventSymbol implementation, BindingDiagnos
         /// </summary>
         protected abstract bool AccessorsHaveImplementation { get; }
 
-        internal bool IsPartialDefinition => IsPartial && !AccessorsHaveImplementation && !HasExternModifier;
+        internal sealed override bool IsPartialDefinition => IsPartial && !AccessorsHaveImplementation && !HasExternModifier;
```

Comment:
>sealed [](http://example.com/codeflow?start=17&length=6)

Consider adding `sealed` to `PartialDefinitionPart` and `PartialImplementationPart` as well.